corehq/apps/domain/views.py

```diff
@@ -1255,12 +1255,10 @@ def create_form(form_class):
     @property
     @memoized
     def select_subscription_type_form(self):
-        if self.request.method == 'POST':
-            for form_slug in self.slug_to_form:
```

Comment:
OK, just checking. It's always weird to stumble upon such "how did this ever work..." kind of code.